### ISSUE TITLE
forge/operations should be POST not GET

### ISSUE THREAD

**Issue by utdrmac** [DefinitelyNotAGoat/go-tezos#24]
Calling CreateBatchPayment() leads to calling forgeOperationBytes() which calls client.GetReponse() https://github.com/DefinitelyNotAGoat/go-tezos/blob/66b1b13b64b1c78053a391c495c0d97c6a70e16f/goTezosNode.go#L44

This is a POST operation, not a GET, therefore the call fails.

```
$ ~/tezos/tezos-client rpc list /chains/main/blocks/head/helpers/forge/operations

Available services:

  - POST /chains/main/blocks/head/helpers/forge/operations
      Forge an operation
```

**Comment by utdrmac** [comment]
Fixed #25

**Comment by DefinitelyNotAGoat** [comment]
Thanks for the fix, I will close this when your pull request gets merged.
